qutebrowser/browser/hints.py

```diff
@@ -748,6 +748,38 @@ def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
         self._context.tab.find_all_elements(selector, self._start_cb,
                                             only_visible=True)
 
+    def _get_visible_hints(self):
+        """Get elements which are currently visible.
+        """
```

Comment:
It's still much faster though (44s instead of almost 3min here) now that I turned it on again :laughing: